nipype/interfaces/fsl/utils.py

```diff
@@ -26,11 +26,56 @@
 from ..base import (traits, TraitedSpec, OutputMultiPath, File,
                     CommandLine, CommandLineInputSpec, isdefined)
 from ...utils.filemanip import (load_json, save_json, split_filename,
-                                fname_presuffix)
+                                fname_presuffix, copyfile)
 
 warn = warnings.warn
 warnings.filterwarnings('always', category=UserWarning)
 
+class CopyGeomInputSpec(FSLCommandInputSpec):
+    in_file = File(exists=True, mandatory=True, argstr="%s", position=0,
+                   desc="source image")
+    dest_file = File(exists=True, mandatory=True, argstr="%s", position=1,
+                     desc="destination image", copyfile=True)
+    ignore_dims = traits.Bool(desc=('Do not copy image dimensions'),
+                              argstr='-d', position="-1")
+
+class CopyGeomOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc="image with new geometry header")
+
+class CopyGeom(FSLCommand):
+    """Use fslcpgeom to copy the header geometry information to another image.
+
+    Copy certain parts of the header information (image dimensions, voxel dimensions,
+    voxel dimensions units string, image orientation/origin or qform/sform info)
+    from one image to another. Note that only copies from Analyze to Analyze
+    or Nifti to Nifti will work properly. Copying from different files will result
+    in loss of information or potentially incorrect settings.
+
+    """
+    _cmd = "fslcpgeom"
+    input_spec = CopyGeomInputSpec
+    output_spec = CopyGeomOutputSpec
+
+    def _run_interface(self, runtime):
```

Comment:
in a workflow there is no issue, as the `copyfile` directive will make a local copy. however, as an interface the python interface should have similar functionality as the original commandline one which is destructive - and we should note it in the docstring as well. the only issue is that since an interface doesn't know if it's running in a workflow, it will still create a copy of the dest file.